src/test/java/util/ConfigLoaderTest.java

```diff
@@ -0,0 +1,37 @@
+package util;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class ConfigLoaderTest {
+
+    private ConfigLoader configLoader;
+
+    @BeforeEach
+    public void setUp() throws IOException {
+        configLoader = new ConfigLoader();
+    }
+
+    @Test
+    public void testLoadPropertyFromFile() {
+        String expectedValue = "someValue";
+        assertEquals(expectedValue, configLoader.getProperty("someKey"));
+    }
+
+    @Test
+    public void testLoadPropertyFromConstructor() {
+        ConfigLoader configLoaderWithProperty = new ConfigLoader("testKey", "testValue");
+        assertEquals("testValue", configLoaderWithProperty.getProperty("testKey"));
+    }
```

Comment:
You could also add a `testLoadNonExistantPropertyFromConstructor`

c = new ConfigLoader("key", "value")
c.getProperty("nonexistant") // should error

Reply:
Yeah, it returns null